cmd/saturn-l2/main.go

```diff
@@ -71,6 +70,10 @@ const (
 	// MAX_CONCURRENT_L1_REQUESTS_VAR configures the environment variable that determines the maximum
 	// number of CAR file requests that will be processed concurrently for a single L1. defaults to 3.
 	MAX_CONCURRENT_L1_REQUESTS_VAR = "MAX_CONCURRENT_L1_REQUESTS"
+
+	// TEST_L1_IPs_VAR is a comma separated list of L1 IP Addresses that this L2 will connect to and serve
+	// CAR files to.
+	TEST_L1_IPs_VAR = "TEST_L1_IPs"
```

Comment:
I wonder if we can support this use case without commenting out a bunch of code, so that we can also use this later on. Could we keep the orchestrator code but bypass it if TEST_L1_IPS exists?

Reply:
```suggestion
	TEST_L1_IPs_VAR = "TEST_L1_IPS"
```

Reply:
Will make the change !